src/styles/stars.css

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-position: center;
+  background-repeat: no-repeat;
+
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+
+  background-image: url("../images/star.svg");
+}
+
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background-image: url("../images/star-active.svg");
+}
+
+.stars--5 :nth-child(-n + 5) {
+  margin-right: 0;
+}
```

Comment:
you can set `margin: 4px` to all stars except the last one using `:not` pseudoclass